### DISCUSSION TITLE
If we ever need a DB how about

### DISCUSSION THREAD

**Discussion by daKmoR** [modernweb-dev/rocket#240] [Ideas]
https://supabase.io/ => an open source firebase clone
https://fauna.com/ => a cloud db

**Comment by bennypowers** [comment]
supabase would be a nice option, but where would this db live and how would it run? so far we're recommending netlify, which AFAIK is very much purely JAMstack - no db

**Reply by daKmoR** [reply]
you query the database via serverless functions

**Reply by daKmoR** [reply]
ur via the build
